Fix Metadata field getting cut off

Arnei opened the pull request:
Some metadata fields would not display their full content even if there was still space available. This patch should help with that.

Before:
<img width="1081" height="533" alt="Bildschirmfoto vom 2025-07-30 09-08-13" src="https://github.com/user-attachments/assets/04794b85-f659-427b-8410-f3e9b6e420e0" />

After:
<img width="1086" height="474" alt="Bildschirmfoto vom 2025-07-30 09-07-54" src="https://github.com/user-attachments/assets/d92efe1e-d1e7-4346-b26b-1094eca34c60" />

### How to test

Check that inputs/textareas in modals do not behave strangely.

Arnei committed:
Fix Metadata field getting cut off

Some metadata fields would not display their full content
even if there was still space available. This patch should help
with that.

gregorydlogan reviewed (changes requested):
Hammering the keyboard and hitting enter randomly looks good on the Metadata tab, but when I get to the summary I see this:

<img width="876" height="155" alt="image" src="https://github.com/user-attachments/assets/4c7c69d4-57f6-45dc-850e-a21147ba89af" />

The data is all there, and the modal will scroll right for this:

<img width="876" height="155" alt="image" src="https://github.com/user-attachments/assets/dc14841d-897e-4e02-ba7c-48ee56c60b7b" />


FTR, the metadata tab looks correct to me:
<img width="831" height="893" alt="image" src="https://github.com/user-attachments/assets/45dd9927-2078-4e0f-9f35-07db29e3c6cc" />

Arnei committed:
Avoid table overflowing in modals

Some table fields in modals may overflow if they contain a
very long word, for example the metadata td's in the create event modal
on the summary page. This commit fixes that by specifying word break
bevahiour for td's.
